src/FourierTransforms.jl/ctfft.jl

```diff
@@ -1,4 +1,13 @@
 using Primes
+using IntervalArithmetic
+import Base: cbrt
+AbstractFloatOrInterval = Union{AbstractFloat, Interval}
+
+# Monkey-patch cbrt for Intervals ... should be done more properly by changing
+# the rounding mode ...
+# TODO Remove this once https://github.com/JuliaIntervals/IntervalArithmetic.jl/issues/310
+#      is dealt with.
+cbrt(i::Interval) = Interval(prevfloat(cbrt(i.lo)), nextfloat(cbrt(i.hi)))
```

Comment:
Remove that (it can go into the test script)

Reply:
Ok, will do.